src/registrar/widgets.py

```diff
@@ -0,0 +1,16 @@
+# widgets.py
+
+from django.contrib.admin.widgets import FilteredSelectMultiple
+from django.utils.safestring import mark_safe
+
+
+class NoAutocompleteFilteredSelectMultiple(FilteredSelectMultiple):
+    """Firefox and Edge are unable to correctly initialize the source select in filter_horizontal
+    widgets. We add the attribute autocomplete=off to fix that."""
+
+    def render(self, name, value, attrs=None, renderer=None):
+        if attrs is None:
+            attrs = {}
+        attrs["autocomplete"] = "off"
+        output = super().render(name, value, attrs=attrs, renderer=renderer)
+        return mark_safe(output)  # nosec
```

Comment:
(Q) Why mark_safe? I think this should be fine as long as there isn't any user-supplied data in here

I'm not too familiar with the structure of the data here (shoot me down), but if any user data is embedded inside of "output" (like options for example), then this should be avoided. For instance, you could create a user with the first name <script (etc)>, or a myriad of other things.

Reply:
Here, we are adding the autocomplete attribute with a value of "off" to the attrs dictionary, which will affect the rendering of the HTML output. The use of mark_safe is necessary, no?